assets/mdn-annotation.css

```diff
@@ -0,0 +1,236 @@
+.mdn {
+    display: block;
+    font: 12px sans-serif;
+    position: absolute;
+    z-index: 9;
+    right: 0.3em;
+    background-color: #eee;
+    margin: -26px 0 0 0;
+    padding: 7px 5px 5px 6px;
+    min-width: 140px;
+    box-shadow: 0 0 3px #999;
+}
+
+.mdn button {
+    cursor: pointer;
+    position: absolute;
+    left: 0;
+    top: -2px;
+    height: 1em;
+    border: none;
+    color: #000;
+    background: transparent;
+    margin-left: -8px;
+    font-size: 10px;
+    line-height: 1em;
+}
+
+.mdn b {
+    color: #fff;
+    background-color: #000;
+    font-weight: normal;
+    font-family: zillaslab, Palatino, "Palatino Linotype", serif;
+    padding: 2px 3px 0px 3px;
+    line-height: 1.3em;
+    vertical-align: top;
+}
+
+.mdn details {
+    display: inline;
+    margin-top: 1px;
+}
+
+.mdn summary a {
+    margin-left: -4px;
+}
+
+.nosupportdata {
+    font-style: italic;
+    margin-top: 4px;
+    margin-left: 8px;
+    padding-bottom: 8px;
+}
+
+.mdnsupport {
+    display: table;
+    margin-top: 4px;
+}
+
+.mdnsupport>span {
+    display: table-row;
+    padding: 0.2em 0;
+    padding-top: 0.2em;
+    font-size: 9.6px;
+}
+
+.mdnsupport>span>span {
+    display: table-cell;
+    padding: 0 0.5em;
+    vertical-align: top;
+    line-height: 1.5em;
+}
+
+.mdnsupport>span>span:last-child {
+    text-align: right;
+    padding: 0;
+}
+
+.mdnsupport>span.no {
+    color: #CCCCCC;
+    filter: grayscale(100%);
+}
+
+.mdnsupport>span.unknown {
+    color: #CCCCCC;
+    filter: grayscale(100%);
+}
+
+.mdnsupport>span.no::before {
+    opacity: 0.5;
+}
+
+.mdnsupport>span.unknown::before {
+    opacity: 0.5;
+}
+
+.mdnsupport>span::before {
+    content: "";
+    display: table-cell;
+    min-width: 1.5em;
+    height: 1.5em;
+    background: no-repeat center center;
+    background-image: none;
+    background-size: auto auto;
+    background-size: contain;
+    text-align: right;
+    font-size: 0.75em;
+    font-weight: bold;
+}
+
+.mdnsupport>.chrome::before {
```

Comment:
Can we use CSS variables here?

Reply:
Like
``` css
.mdnsupport > .chrome {
  --bg: url(https://resources.whatwg.org/browser-logos/chrome.svg);
}
```

Reply:
Will change. I copied all the css from the original HTMLWG repo and just kept their version.

Reply:
Checked again. Maybe I'll just group the selector without adding variables, like
```css
.mdnsupport > .chrome,
.mdnsupport>.chrome_android::before {
  --bg: url(https://resources.whatwg.org/browser-logos/chrome.svg);
}
```

Reply:
I wonder if we should link https://resources.whatwg.org/standard-shared-with-dev.css instead? Then we don't need to worry about maintaining the style and it will always match HTML?